DataGrid: Add (Update) AI Assistant Demo Description (#33590)
DevExpress/DevExtreme: apps/demos/Demos/DataGrid/AIAssistant/description.md

The DevExtreme [DataGrid](https://js.devexpress.com/Documentation/Guide/UI_Components/DataGrid/Overview/) component ships with our AI Assistant feature. This capability allows you to interact with the component using natural language. In this demo, the AI Assistant is enabled for a DataGrid that displays mock sale data with over 1500 records.
<!--split-->

[note]

AI services used for this demo have been rate and data limited. As such, you may experience performance-related delays when exploring the capabilities of the DataGrid AI Assistant.

When connected to your own AI model/service without rate and data limits, the AI Assistant will perform seamlessly, without artificial delays. Note that DevExtreme does not offer an AI REST API and does not ship any built-in LLMs/SLMs.

[/note]

The AI Assistant supports DataGrid features such as the following:

- [Filtering and Searching](/Documentation/Guide/UI_Components/DataGrid/Filtering_and_Searching/)
- [Sorting](/Documentation/Guide/UI_Components/DataGrid/Sorting/)
- [Grouping](/Documentation/Guide/UI_Components/DataGrid/Grouping/)

> - [Paging](/Documentation/Guide/UI_Components/DataGrid/Paging/)

Reviewer: There are much more functionality. Here is the demo help text:

💡 The DataGrid AI Assistant allows you to control the component using natural language. You can execute commands such as the following:
  • Sort records
  • Apply a filter
  • Search for a specific value
  • Group records by a field
  • Focus and select rows
  • Modify paging settings
  • Pin, resize, and reorder columns
  • Configure data summaries
  • Pick a suggestion or enter a custom request to get started.

Please contact @dmirgaev to ask if we need even add something.

These capabilities are enabled in this demo. You can use/configure these features in the AI Assistant chat.

This demo also configures [suggestions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#suggestions) for the AI Assistant chat. These buttons allow you to interact with the assistant in one click using pre-defined prompts. For additional information about suggestions, refer to the following technical demo: [DevExtreme Chat - Prompt Suggestions](/Demos/WidgetsGallery/Demo/Chat/PromptSuggestions/).

> The AI Assistant capability is also available for the DevExtreme [TreeList](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#aiAssistant) component.

Reviewer: We could announce TreeList AI Assistant in the beginning of the description. Now it looks like a post scriptum...

> The DevExtreme [DataGrid](https://js.devexpress.com/Documentation/Guide/UI_Components/DataGrid/Overview/) AI Assistant allows you to interact with the component using natural language. You can use/configure multiple DataGrid features in the AI Assistant chat using written prompts. The AI Assistant supports DataGrid features such as:

Reviewer 2: ```suggestion
The AI Assistant for the DevExtreme [DataGrid](https://js.devexpress.com/Documentation/Guide/UI_Components/DataGrid/Overview/) allows you to interact with the component using natural language. The chat also supports [speech-to-text input](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#speechToTextEnabled), ideal for hands-free interactions or entering longer prompts.

 You can use/configure the following DataGrid features in the AI Assistant chat prompts:
```

- [Filtering and Searching](/Documentation/Guide/UI_Components/DataGrid/Filtering_and_Searching/)
- [Sorting](/Documentation/Guide/UI_Components/DataGrid/Sorting/)
- [Grouping](/Documentation/Guide/UI_Components/DataGrid/Grouping/)
- [Paging](/Documentation/Guide/UI_Components/DataGrid/Paging/)

> - [Row Focus](/Documentation/Guide/UI_Components/DataGrid/Focused_Row/)

Reviewer 2: ```suggestion
- [Focused Row](/Documentation/Guide/UI_Components/DataGrid/Focused_Row/)
```

- [Selection](/Documentation/Guide/UI_Components/DataGrid/Selection/)
- [Summaries](/Documentation/Guide/UI_Components/DataGrid/Summaries/Predefined_Aggregate_Functions/)
- [Column Fixing](/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Fixing/), [Resizing](/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Sizing/), and [Reordering](/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Reordering/)

> The chat also supports [speech-to-text input](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#speechToTextEnabled), ideal for hands-free interactions or entering longer prompts.

Reviewer 2: ```suggestion
```

The AI Assistant feature is also available for the DevExtreme [TreeList](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#aiAssistant) component.
<!--split-->

> In this demo, the AI Assistant is enabled for a DataGrid that displays mock sale data with over 1500 records.

Reviewer 2: ```suggestion
In this demo, the AI Assistant is enabled for a DataGrid that displays mock sales data with over 1500 records.
```

[note]

AI services used for this demo have been rate and data limited. As such, you may experience performance-related delays when exploring the capabilities of the DataGrid AI Assistant.

When connected to your own AI model/service without rate and data limits, the AI Assistant will perform seamlessly, without artificial delays. Note that DevExtreme does not offer an AI REST API and does not ship any built-in LLMs/SLMs.

[/note]

> To enable the AI Assistant, configure [aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#aiIntegration) or **aiAssistant**.[aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/aiAssistant/#aiIntegration) and set **aiAssistant**.[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/aiAssistant/#enabled) to `true`.

Reviewer 2: ```suggestion
To enable the AI Assistant, configure the [aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#aiIntegration) or **aiAssistant**.[aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/aiAssistant/#aiIntegration) object and set the **aiAssistant**.[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/aiAssistant/#enabled) property to `true`. Once activated, the DataGrid adds a predefined item (*"aiAssistantButton"*) to the toolbar. This button opens the AI Assistant chat in a draggable pop-up window.
```

> When this feature is enabled, DataGrid adds a predefined item (*"aiAssistantButton"*) to the component's toolbar. This button opens the AI Assistant chat. The assistant is displayed in a popup that users can move around the screen as needed.

Reviewer 2: ```suggestion
```

> This demo also configures [suggestions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#suggestions) for the AI Assistant chat. These buttons allow you to interact with the assistant in one click using pre-defined prompts. For additional information about suggestions, refer to the following technical demo: [DevExtreme Chat - Prompt Suggestions](/Demos/WidgetsGallery/Demo/Chat/PromptSuggestions/).

Reviewer 2: ```suggestion
This demo also configures [suggestions](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#suggestions) for the AI Assistant chat. These buttons allow you to interact with the assistant in one click using pre-defined prompts. For additional information about suggestions, refer to the following demo: [DevExtreme Chat - Prompt Suggestions](/Demos/WidgetsGallery/Demo/Chat/PromptSuggestions/).
```
